src/main/java/rx/Single.java

```diff
@@ -1270,6 +1271,62 @@ public R call(Object... args) {
     }
 
     /**
+     * Returns a Single that subscribes to this Single lazily, caches its next event
```

Comment:
"Returns a Single that subscribes to a this Single lazily, caches its success or error event"